### ISSUE TITLE
Torrent and ocamlfuse

### ISSUE THREAD

**Issue by zerfeus** [astrada/google-drive-ocamlfuse#366]
I want to send my torrents directly to a folder on my google drive with ocamlfuse, is this possible? I configured, but the torrent client stops downloading some files and does not come back. I would be grateful if anyone can help me with some confiration.
This is my config:
`apps_script_format=json
apps_script_icon=
async_upload=true
cache_directory=
client_id=494147679546-uurlhrr3mlvrs0gv0aof8irl2521b3l4.apps.googleusercontent.$
client_secret=sEudvWW6g4Gvgp3tfAtUUipk
connect_timeout_ms=5000
curl_debug_off=false
data_directory=
delete_forever_in_trash_folder=false
docs_file_extension=true
document_format=odt
document_icon=
download_docs=true
drawing_format=png
drawing_icon=
form_format=zip
form_icon=
fusion_table_format=desktop
fusion_table_icon=
keep_duplicates=false
large_file_read_only=false
large_file_threshold_mb=100000
log_directory=
lost_and_found=false
low_speed_limit=0
low_speed_time=0
map_format=desktop
map_icon=
max_cache_size_mb=200000
max_download_speed=0
max_memory_cache_size=10485760
max_retries=8
max_upload_chunk_size=1099511627776
max_upload_speed=0
memory_buffer_size=100048576
metadata_cache_time=36000
presentation_format=pdf
presentation_icon=
read_ahead_buffers=5
read_only=false
root_folder=
shared_with_me=false
spreadsheet_format=ods
spreadsheet_icon=
sqlite3_busy_timeout=5000
stream_large_files=false
umask=0o002
verification_code=`

**Comment by astrada** [comment]
No, downloading torrents to the mount-point is not really supported because Google Drive API doesn't work well with file chunks.

**Comment by zerfeus** [comment]
Does it work well if you move full heavy files (already downloaded)?

**Comment by astrada** [comment]
Yes, but I would copy the files instead of moving them, to be sure that they are uploaded without errors (then you can remove the old files).
